doc/developer_guide/developer-guide.md

```diff
@@ -2,74 +2,18 @@
 
 ## How to Release?
 
-Creating a new release of the NC requires 4 steps
+Creating a new release of the NC requires 3 steps
 
-1. [Prepare the release](#prepare-the-release)
+1. Prepare the release
 2. Merge your pull request to the default branch
-3. [Switch to main branch and update it from remote](#switching-to-the-default-branch)
```

Comment:
This is PTB default now and should only link to PTB User Guide whenever possible.